unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -45,33 +46,29 @@
     6, 6, 6, 6};
 
   public static UTF8String fromBytes(byte[] bytes) {
```

Comment:
Maybe it's not necessary, but should we add a comment to call out the fact that this doesn't defensively copy `bytes`?